src/wazuh_modules/inventory_harvester/tests/CMakeLists.txt

```diff
@@ -37,5 +31,16 @@ add_executable(inventory_harvester_utest
     ${UNIT_SRC_FILES}
 )
 
-target_link_libraries(inventory_harvester_utest gtest gtest_main gmock)
+target_link_libraries(inventory_harvester_utest
+    debug gtestd
+    debug gmockd
+    debug gtest_maind
+    debug gmock_maind
+    optimized gtest
+    optimized gmock
+    optimized gtest_main
+    optimized gmock_main
+)
+
 target_include_directories(inventory_harvester_utest PRIVATE ${TEST_SRC_DIR} ${TEST_MOCK_DIR})
```

Comment:
If you use include_directories, this is reduntant. I'll avoided using include_directories since I think the goal is to use target_include_directories instead based on the base implementation by Octavio.